localflavor/mx/forms.py

```diff
@@ -163,6 +163,50 @@ def _has_inconvenient_word(self, rfc):
         return first_four in RFC_INCONVENIENT_WORDS
 
 
+class MXCLABEField(RegexField):
+    """This field validates a CLABE (Clave Bancaria Estandarizada).
+
+    A CLABE is a 18-digits long number. The first 6 digits denote bank and branch number.
+    The remaining 12 digits denote an account number, plus a verifying digit.
+
+    More info:
+    https://es.wikipedia.org/wiki/CLABE
+
+    .. versionadded:: 1.3
+    """
+
+    default_error_messages = {
+        'invalid': _('Enter a valid CLABE.'),
+        'invalid_checksum': _('Invalid checksum for CLABE.'),
+    }
+
+    def __init__(self, min_length=18, max_length=18, *args, **kwargs):
+        clabe_re = r'^\d{18}$'
+        clabe_re = re.compile(clabe_re)
+        super(MXCLABEField, self).__init__(clabe_re, min_length, max_length, *args, **kwargs)
+
+    def _checksum(self, value):
+        verification_digit = int(value[-1])
+        number = value[:-1]
+
+        ponderator = (3, 7, 1, 3, 7, 1, 3, 7, 1, 3, 7, 1, 3, 7, 1, 3, 7)
```

Comment:
It would be better to use `weights` or `weight_factors` for this variable instead of `ponderator`.